src/main/java/net/minecraftforge/fluids/BlockFluidBase.java

```diff
@@ -90,11 +90,11 @@
     public static final PropertyInteger LEVEL = PropertyInteger.create("level", 0, 15);
     public static final PropertyFloat[] LEVEL_CORNERS = new PropertyFloat[4];
     public static final PropertyFloat FLOW_DIRECTION = new PropertyFloat("flow_direction");
-    public static final IUnlistedProperty[] FLUID_RENDER_PROPS;
+    public static final IUnlistedProperty<Float>[] FLUID_RENDER_PROPS;
 
     static
     {
-        ImmutableList.Builder<IUnlistedProperty> builder = ImmutableList.builder();
+        ImmutableList.Builder<IUnlistedProperty<Float>> builder = ImmutableList.builder();
```

Comment:
This is potentially a breaking change for existing mods, however I assume no mod will have a problem with being told that they are all float values.